src/blogs/repeatable-openebs-mayastor-deployments-and-benchmarks.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Repeatable OpenEBS Mayastor deployments and benchmarks
```

Comment:
Please check with @AVRahul and fix on these metadata. We need to make this uniform across blogs for maintainability and predictability.

Reply:
Added same metadata as per Rahul's PR